src/artic/impl/bsdf/diffuse.art

```diff
@@ -1,4 +1,5 @@
 // Creates a purely two-sided diffuse BSDF
+// note from Eric: isn't kd in make_bsdf_sample() wrong (no pdf and cos applied)?
```

Comment:
The weight returned by the make_bsdf_sample is already divided by the pdf. The sampler of the diffuse bsdf has cos/pi. Dividing by it returns only kd.

Reply:
In orennayar habe ich es einfach nicht selbst ausgerechnet. In diffuse habe ich die optimierung einfach selber gemacht.